Duplicati/Server/webroot/ngax/templates/commandline.html

```diff
@@ -5,34 +5,34 @@
         <div ng-hide="EditUriState">
 
             <div class="input mixed">
-                <label for="command">Command</label>
+                <label for="command" translate>Command</label>
                 <select name="command" id="command" ng-model="Command" ng-options="item for item in SupportedCommands">
                 </select>
                 <div style="float: left">{{CommandHelp[Command]}}</div>
```

Comment:
I noticed this seemed to have stopped working by https://github.com/duplicati/duplicati/commit/e4f1d4106a99db724d105c88b4e88409f758158b#diff-0911ce5645bee850ccb8f6b5b2a685dbdc1059c105e0d561dbb38461bff4bc7fL75 and I was wondering if this can be safely removed.

Reply:
It looks like it has not been working since 2017, so I would say it is safe to remove :)

Reply:
The PR was created to address this: https://github.com/duplicati/duplicati/pull/5389